src/vireshJJun20/ConvertCase.java

```diff
@@ -0,0 +1,67 @@
+/*
+ Assignement 29 : 17th July 2020
+
+WAP to convert case in all elements of array. 
+Input : 
+String[] array = {"AaVruti", "NiKhil", "AdiTi", "MAhesh", "ArchaNA", "jaGAdeesh"};
+
+Output:
+array = {"aAvRUTI", "nIkHIL","aDItI", "maHESH","aRCHAna","JAgaDEESH"};
+
+*/
+
+package vireshJJun20;
+
+public class ConvertCase {
+
+	static String tempString2 = "";
+
+	// Method accepts Array, converts each index to appropriate case and then saves
+	// in output array.
+	String[] StringConvertCase(String[] inputArray) {
+		String[] outputArray = inputArray;
```

Comment:
If you are creating a new array then you should create till it's length String[] outputArray = new String[inputArray.length], here you are assigning old array in new then what's the difference?